core-bundle/src/Resources/contao/dca/tl_user.php

```diff
@@ -890,7 +890,7 @@ public function checkAdminStatus($varValue, DataContainer $dc)
 	{
 		if (!$varValue && $this->User->id == $dc->id)
 		{
-			$varValue = '1';
+			$varValue = true;
```

Comment:
since this is a `save_callback`, it should probably set an integer value?

Reply:
It will automatically be converted to `int` by Contao anyway.